CMakeLists.txt

```diff
@@ -111,7 +111,15 @@ add_subdirectory(deps/libsdptransform "${CMAKE_CURRENT_BINARY_DIR}/libsdptransfo
 # Add some compile flags to our source files.
 if(MSVC)
 	set_source_files_properties(${SOURCE_FILES}
-		PROPERTIES COMPILE_FLAGS "/W3 /WX")
+		PROPERTIES COMPILE_FLAGS "/W3")
+	target_link_libraries(${PROJECT_NAME} PRIVATE
```

Comment:
Why are these additions needed here. Shouldn't they go on your application's CMakeLists instead of in libmediasoupclient ones?

Reply:
For the compile flags, I simply removed /WX since compiling on Windows generates ~80 warnings, but no errors, and builds fine with /WX disabled. I had only tested the build with those libraries linked, but testing again without them shows they are no longer necessary. They've been removed.

Reply:
ATM I don't have a Windows machine handy, but I would prefer to keep the warnings so they are 'fixed' when possible, or by some contributor.

Reply:
Yes they are still enabled, /W3 is the warning level (unchanged), /WX treats warnings as errors. Hopefully, if I get a chance soon, I'll try to go over the warnings and fix them, since they mostly warn about unused/unreferenced variables